<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Record</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="recnum">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>600</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>158</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save data to database&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close rhis dialog&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="mediaBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>120</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Media Center</string>
   </property>
   <widget class="QRadioButton" name="kodiButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kodi media center&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Kodi</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="spmcButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SPMC media center&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>SPMC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="otherButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>68</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fill in file path and package name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>248</width>
     <height>270</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="versionLabel">
      <property name="text">
       <string>adbLink version: 9999</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Kodi OS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ostypeBox">
        <item>
         <property name="text">
          <string>Android</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Windows</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>macOS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linux</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="devlab">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="description">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a general description for your device. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter or Paste IP or USB Address&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="daddr">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the serial number or IP address for your device. You may also enter a device name if you have one set up with your router.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Kodi partition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="data_root"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Package name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="packagename">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Package name for your Media Center  software&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="filepathButton">
        <property name="text">
         <string>File Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filepath">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the file path for your Media Center.  This is combined with Package Name, e.g.  org.xbmc.kodi/files/.kodi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pfolderButton">
        <property name="toolTip">
         <string>Choose folder for pulled files</string>
        </property>
        <property name="text">
         <string>Pull Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pulldir">
        <property name="toolTip">
         <string>Pulled files are put in this folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>154</width>
     <height>264</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Android partitions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listkodirootBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Double-click to change Kodi root location. The default is /sdcard/. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    border-color: black;
  
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Connected devices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listDevicesp">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Double-click to copy value to address field&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>125</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QCheckBox" name="disableroot">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Disable root for misbehaving su installations&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Disable root</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isusb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>USB Connection</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="plabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="port">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>21</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Port for IP connections. Defaults to 5555&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ostypeBox</tabstop>
  <tabstop>description</tabstop>
  <tabstop>daddr</tabstop>
  <tabstop>data_root</tabstop>
  <tabstop>packagename</tabstop>
  <tabstop>filepath</tabstop>
  <tabstop>pulldir</tabstop>
  <tabstop>isusb</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>filepathButton</tabstop>
  <tabstop>pfolderButton</tabstop>
  <tabstop>spmcButton</tabstop>
  <tabstop>kodiButton</tabstop>
  <tabstop>otherButton</tabstop>
  <tabstop>listkodirootBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
